.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Java CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        java: [8]
+      fail-fast: false
+      max-parallel: 4
```

Comment:
Without a valid matrix this is pointless too. It seems it's a copy&paste leftover